include/world_builder/utilities.h

```diff
@@ -65,8 +65,23 @@ namespace WorldBuilder
     double
     signed_distance_to_polygon(const std::vector<Point<2> > &point_list_,
                                const Point<2> &point_);
+    /**
+     * Takes the list of the fault coordinates and computes a bounding box
+     * based on the minimum and maximum x,y values in the list, and an
+     * additional buffer zone computed using the maximum fault length.
+     * Currenly, only implemented for the spherical coordinates.
+     */
+    std::pair<Point<2>, Point<2> >
+    get_bounding_box (const std::vector<Point<2> > &point_list,
+                      std::size_t original_number_of_coordinates,
```

Comment:
what does this parameter correspond to?

Reply:
I think you could drop this argument and just use `point_list.size()` everywhere

Reply:
Oh yes, thank you! :)